rtp.c

```diff
@@ -689,7 +694,12 @@ void janus_rtp_header_update(janus_rtp_header *header, janus_rtp_switching_conte
 		}
 		/* Compute a coherent timestamp and sequence number */
 		context->a_prev_ts = context->a_last_ts;
-		context->a_last_ts = (timestamp-context->a_base_ts) + context->a_base_ts_prev;
+		if(timestamp >= context->a_base_ts) {
```

Comment:
This is not correct.
Any packet received with a TS lesser than the base ts will have its TS replaced with a value computed on the real time (`janus_get_audio_time_diff`).
E.g. a SSRC change imposes a new base ts with a high value (say 4000000000). Once the ts warps around, any new packet until 4000000000 will have its TS replaced with a new computed one.

Reply:
When SSRC is changed, context->a_base_ts is set to the current timestamp. So, it will not be replaced with a new estimated timestamp. But, this is irrelevant now because we will move this code to the SIP plugin. :)

Reply:
The RTP TS will eventually wrap around (keeping the same SSRC). So after a certain amount of time you'll receive a TS lesser than the last known base TS.

Reply:
Yes, you are right, thanks.